client/scripts/controllers/server/threads.ts

```diff
@@ -566,6 +566,41 @@ class ThreadsController {
     return !(threads.length < DEFAULT_PAGE_SIZE);
   }
 
+  public async getRecentThreads(options: {
+    chainId: string,
+    communityId: string,
+  }) {
+    const { chainId, communityId } = options;
+    const params = {
+      chain: chainId,
+      community: communityId,
+      cutoff_date: moment(Date.now() - (30 * 24 * 3600 * 1000)).toISOString(),
+    };
+    const response = await $.get(`${app.serverUrl()}/bulkThreads`, params);
+    if (response.status !== 'Success') {
+      throw new Error(`Unsuccessful refresh status: ${response.status}`);
+    }
+
+    response.result.comments.forEach((comment) => {
```

Comment:
why do we need to add comments to the store within the threads controller?

Reply:
Because we use comments to determine recent activity!

Reply:
that makes sense for now, but I think we can do something where we update recent activity server-side on the thread model itself when a comment is made. that way we can reduce the work that the browser has to do to process the data needed to render the page.